src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2058,34 +2129,46 @@
           short-summary: Name of resource group. Required when enabling new regions.
           long-summary: When a previously disabled region is enabled to use Network Watcher, a
             Network Watcher resource will be created in this resource group.
+    examples:
+        - name: Configure Network Watcher in a region
```

Comment:
This example does not demonstrate anything useful and does not match the description (it does not "configure" anything, just enables it.) It should be improved or removed.

Reply:
This command is 100% crucial to being able to use network watcher at all. The CLI abstracts the away the weirdness of the network watcher service, so I would say this example is completely appropriate. I would say the title could be improved to something like: "Configure network watcher for use on resources in West US."

Reply:
Do CLI users know about this "weirdness"? All they see is required arguments listed in the command help. This example just says "use the required arguments" which we have agreed is not useful for examples.

Reply:
Resource group is not required though, unless you are enabling regions, which is what the example demonstrates.

Reply:
OK.